.github/workflows/e2e.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       # core git ref should be latest commit for stable soroban functionality
       # the core bin can either be compiled in-line here as part of ci, 
-      SYSTEM_TEST_CORE_GIT_REF: https://github.com/stellar/stellar-core.git#soroban-preview-9-May22
+      SYSTEM_TEST_CORE_GIT_REF: https://github.com/stellar/stellar-core.git#875f47e247cefc25c8a4b3982ee65610e6a620e3
```

Comment:
I think `SYSTEM_TEST_GIT_REF` above needs to be updated to v1.0.16 after stellar/system-test#63, but it looks like the [push was skipped](https://github.com/stellar/system-test/actions/runs/5526018702/jobs/10080448430)? I don't know how the releases work around this.

Reply:
I can assist on sys tests, probably best to skip trying to fix the e2e failures here, so this can get merged first, and then debug from `main` when it's release ready, have learned from past it's not worth trying to fix the tests any sooner.

system-test releases are just a GH release with a [CHANGELOG.md](https://github.com/stellar/system-test/blob/master/CHANGELOG.md) entry. The tag(or any other git ref) is all that's required to use it from other projects.